hubspot3/products.py

```diff
@@ -12,7 +12,6 @@
 class ProductsClient(BaseClient):
     """
     Products extension for products API endpoint
-    THIS API ENDPOINT IS ONLY A PREVIEW AND IS SUBJECT TO CHANGE
```

Comment:
Is this still the case? I didn't see this warning on the product documentation

Reply:
Looks like they've updated that - good catch!